Program.cs

```diff
@@ -1,7 +1,40 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter Password: ");
+         string Password = Console.ReadLine ();
+         if (check (Password) == "true") Console.WriteLine ("Strong password.");
+         else Console.WriteLine ("Not a strong password. \n" + check (Password)) ;
+      }
+      static string check (string pass) {
+         string lower = "", upper = "", digit = "", splchar = "", notstrong = "";

```

Comment:
Can you think of better **data type** rather than **string**?

Reply:
Instead of string used bool.